diff --git a/dist/changesets-changelog-github.cjs.js b/dist/changesets-changelog-github.cjs.js
index f0ab3c46b9cd516ca5635e2e4dab079bac2908a6..7a2c7bf4e6dc32e1e958f65ce78afc1977b2d790 100644
--- a/dist/changesets-changelog-github.cjs.js
+++ b/dist/changesets-changelog-github.cjs.js
@@ -134,8 +134,8 @@ const changelogFunctions = {
       };
     })();
     const users = usersFromSummary.length ? usersFromSummary.map(userFromSummary => `[@${userFromSummary}](https://github.com/${userFromSummary})`).join(", ") : links.user;
-    const prefix = [links.pull === null ? "" : ` ${links.pull}`, links.commit === null ? "" : ` ${links.commit}`, users === null ? "" : ` Thanks ${users}!`].join("");
-    return `\n\n-${prefix ? `${prefix} -` : ""} ${firstLine}\n${futureLines.map(l => `  ${l}`).join("\n")}`;
+    const suffix = [links.pull === null ? "" : ` ${links.pull}`, links.commit === null ? "" : ` ${links.commit}`, users === null ? "" : ` (${users})`].join("");
+    return `\n\n- ${firstLine}${suffix ? ` - ${suffix}` : ""}\n${futureLines.map(l => `  ${l}`).join("\n")}`;
   }
 };
 
